helpdesk_mgmt/readme/CONTRIBUTORS.rst

```diff
@@ -20,6 +20,9 @@
 * `Factor Libre <https://factorlibre.com>`_:
 
   * María Alhambra
+  * Daniel Cano
```

Comment:
How can it be that adding a simple method supposes 3 new contributors?

Reply:
Because we did some work on the base but forgot to add ourselves as contributors.

Reply:
Then please add separately in another PR and document that linking to the commits. I remember a commit from you doing the translation, which applies for the contributor role, but not from the others (but I haven't remembered in deep).

Reply:
I did an early translation and mail templates along with Aitor. I will tell the rest to do a PR as you say.